### PR TITLE
chore: move client to new repository

### PR DESCRIPTION
**Pull request by skeggse**
[See new repo](https://github.com/mixmaxhq/publication-client).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,3 +23,18 @@
 * 1.6.0 Pin the uws version.
 
 * 1.5.1 Fix Subscription initialization parameters
+
+* 1.5.1 Prevent `Subscription#whenReady` from resolving prematurely after the websocket connects; implement `PublicationClient#whenConnected`
```
Comment (wearhere) [inline CHANGELOG.md]: Could you dedupe this bullet with the one above?

File: README.md [hunk 2]
```diff
@@ -140,7 +140,6 @@ May contain some client releases pre-1.5.0.
 * 1.5.0 Use Yarn; fix `uws` dependency after previous versions were unpublished.
 * 1.4.5 `err._publicationName` -> `err.publicationName`
 * 1.4.4 Add the publication name to the error we emit on subscription errors.
-* 1.4.3 Delete subscription references client side when we unsubscribe from a publication.
```
Comment (wearhere) [inline README.md]: Did this release not exist?

File: README.md [hunk 3]
```diff
@@ -140,7 +140,6 @@ May contain some client releases pre-1.5.0.
 * 1.5.0 Use Yarn; fix `uws` dependency after previous versions were unpublished.
```
Comment (wearhere) [inline README.md]: thoughts on removing this changelog from the README now that we have the separate file?

Comment (wearhere) [inline README.md]: Can we link to the client somewhere in the README?

**Comment by wearhere** [comment]
Can we document the reasons for moving the client to a separate repo? (Easier to find / version?)

File: README.md (hunk 2 above)
Comment (skeggse) [reply README.md]: It was a client-only release, and I moved the notes to that repo

**Pull request merged**